db/migrate/20181130111723_add_volumes.rb

```diff
@@ -0,0 +1,55 @@
+class AddVolumes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :volumes, :id => :bigserial do |t|
+      t.references :tenant, :type => :bigint, :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
+      t.references :source, :type => :bigint, :index => false, :null => false, :foreign_key => {:on_delete => :cascade}
+      t.references :source_region, :type => :bigint, :index => true, :foreign_key => {:on_delete => :cascade}
+
+      t.string :source_ref
+      t.string :name
+      t.string :status
+      t.bigint :size
+      t.jsonb  :extra
+
+      t.timestamps
+      t.datetime   :archived_on
```

Comment:
Can you update this to archived_at

Reply:
of course, sorry I forgot I've already merged your PR :-D